pages/changelog.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <main class="changelog-page">
+    <NavigationBar title="Changelog" />
+    <div class="content formatted">
+      <h1>{{ $t("navbar.changelog") }}</h1>
+
+      <div class="grid">
+        <div v-for="change in changes.versions" :key="change.name" class="versionEntry">
+          <article>
+            <div>
+              <h3>{{ change.name }}</h3>
+              <small>{{ change.date }}</small>
```

Comment:
Please localize the date.
Maybe use something like this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString

Reply:
I will work on it if i find time.